labshare-client/src/components/RegistrationSections/LoginInformation.vue

```diff
@@ -1,4 +1,13 @@
-
+<i18n>
+    {
+    "en":{
+    "loginInfo": "Login information"
```

Comment:
Both should be capitalised here (in which case it matches the German exactly, which means it is technically redundant, but omitting it might be too obscure for future devs so its presence arguably has syntactic value)